src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.model;
+
+public class User {
+    private int id;
+    private String name;
+    private String lastName;
+    private String email;
```

Comment:
let's keep it simple in this task and not have the id for now
in future, all our entities will have an id
```suggestion
    private String name;
    private int age;
```

Reply:
done